src/main/java/uk/gov/ons/ctp/response/sample/service/impl/SampleServiceImpl.java

```diff
@@ -227,28 +227,21 @@ public Integer initialiseSampleUnitsForCollectionExcerciseCollection(UUID sample
   }
 
   @Override public SampleSummary ingest(MultipartFile file, String type) throws Exception {
-
-    SampleSummary sampleSummary = new SampleSummary();
-
     switch (type) {
-    case "bres":
-      sampleSummary = csvIngesterBusiness.ingest(file);
-      break;
-    case "census":
-      sampleSummary = csvIngesterCensus.ingest(file);
-      break;
-    case "social":
-      sampleSummary = csvIngesterSocial.ingest(file);
-      break;
+      case "B":
```

Comment:
type.toLowerCase() is being called, this needs to be changed to "b". Should fix the travis build.

Reply:
Nice one, fixed it Friday but forgot to push it up :)